torchgeo/transforms/indices.py

```diff
@@ -8,16 +8,16 @@
 - https://github.com/davemlz/awesome-spectral-indices
 """
 
-from typing import Dict
+from typing import Dict, Optional
 
 import torch
+from kornia.augmentation import IntensityAugmentationBase2D
 from torch import Tensor
-from torch.nn.modules import Module
 
 _EPSILON = 1e-10
 
 
-class AppendNormalizedDifferenceIndex(Module):
+class AppendNormalizedDifferenceIndex(IntensityAugmentationBase2D):
```

Comment:
What is this super class?

Reply:
Nevermind https://kornia.readthedocs.io/en/latest/augmentation.base.html#kornia.augmentation.IntensityAugmentationBase2D